xcvm/cosmwasm/contracts/asset-registry/src/contract.rs

```diff
@@ -0,0 +1,168 @@
+use std::collections::BTreeMap;
+
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{to_binary, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult};
+
```

Comment:
nit: I personally prefer no linebreaks between imports, as then they will be sorted by rustfmt